packages/cozy-sharing/package.json

```diff
@@ -32,7 +32,8 @@
     "cozy-doctypes": "^1.82.2",
     "lodash": "^4.17.19",
     "react-autosuggest": "^9.4.3",
-    "react-tooltip": "^3.11.1"
+    "react-tooltip": "^3.11.1",
+    "snarkdown": "2.0.0"
```

Comment:
we should not fix the version

Reply:
Fixed. I copy-pasted that line from `cozy-drive`, should I do the fixe on this project too?